test/cppsuite/tests/test_live_restore.cpp

```diff
@@ -332,10 +356,19 @@ main(int argc, char *argv[])
 
     auto crud_session = connection_manager::instance().create_session();
 
-    if (!fresh_start)
+    if (!fresh_start && !background_debug)
         configure_database(crud_session);
 
-    do_random_crud(crud_session, fresh_start);
+    if (background_debug) {
+        // Loop until the state stat is complete!
+        int64_t state = 0;
+        while (state != WT_LIVE_RESTORE_COMPLETE) {
```

Comment:
Since `threads_max=0` as it stands, this will never be the case.

Reply:
Yeah that's intentionally hardcoded until we have better options parsing and concurrency.

Reply:
For reference: https://jira.mongodb.org/browse/WT-13889